NEWS.md

```diff
@@ -10,6 +10,9 @@
 * `smape()` is a numeric metric that is based on percentage errors 
 (@riazhedayati, #67).
 
+* `conf_mat()` got two `ggplot2::autoplot()` methods for easy visualization 
+(@EmilHvitfeldt, $10).
```

Comment:
`$10`? did you mean `#80`? 🤣

Reply:
I think I would say:

`conf_mat` objects now have

rather than

`conf_mat()` got

Reply:
Should I also reference #10?

Reply:
oh yes please

Reply:
reference only 10, not 80. sorry

Reply:
No problem, I was the one using $ instead of # 😄